src/web/mjs/connectors/SoftKomik.mjs

```diff
@@ -27,16 +27,22 @@ export default class SoftKomik extends Connector {
         let mangaList = [];
         for (let page = 1, run = true; run; page++) {
             const mangas = await this._getMangasFromPage(page);
-            mangas.length > 0 ? mangaList.push(...mangas) : run = false;
+            if(mangas.length == 0) {
```

Comment:
Is it possible to include this check into `_getMangasFromPage` and return an empty manga list instead a duplicate of the last page list (e.g. by checking the page exceeds the max. number of pages or something)

Reply:
Dont think so, when the api reach the last page, it will repeat the last page. Even the website itself do not have a max page number and will repeat the managa as seen in the img below. The only way is to check for duplicate.
![image](https://user-images.githubusercontent.com/9015764/133419059-7818a76e-4b3b-415f-b398-ccba4254db2c.png)